doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -0,0 +1,144 @@
+.. _bluetooth_mesh_brg_cfg:
+
+Subnet Bridge
+#############
+
+The Bluetooth Mesh Subnet Bridge feature allows a node to relay messages between different subnets
+in a Mesh network.
+
+The Subnet Bridge feature includes two models:
+
+.. toctree::
+   :maxdepth: 1
+
+   brg_cfg_srv
+   brg_cfg_cli
+
+The Bridge Configuration Server model is mandatory for supporting the Subnet Bridge feature.
+The Bridge Configuration Client model is optional and allows nodes to configure the Subnet Bridge on
+other nodes.
+These models define the necessary states, messages, and behaviors for configuring and managing the
+Subnet Bridge feature.
+
+Concepts
+========
+
+The Subnet Bridge feature introduces several concepts to enable message relaying between different
+subnets in a Mesh network.
```

Comment:
To get a better understanding of the Subnet Bridge feature and its capabilities, an overview of a few concepts is needed.